### ISSUE TITLE
Code Release Plan

### ISSUE THREAD

**Issue by MinJunKang** [IGL-HKUST/DiffusionAsShader#1]
Hello, thanks for sharing wonderful work!

I am looking forward to running your model! 
When will this project be open-sourced? I am eager to start my new research project based on your work.

Also, I have simple questions about your work.

1. In your paper, 8 H800 (80GB) are used to train the model with batchsize of 64. I can use 8 A6000 of smaller VRAM (48GB). Can I train your model with my GPU settings (maybe using smaller batchsize). What will be the possible problems?
2. I noticed that most of the cases of demos in the paper are conditioned by metric-scaled depths. If I use relative-scaled monocular depth instead, is it still plausible?
3. For the camera control example, does Das work with drastic changes of rotation? etc) object-centric 360-degree circle rotation.

Thanks

**Comment by RuihanLu** [comment]
Similar question here. How does DaS perform on large-scale, long-term camera movement tasks? The current demos only give examples of short distance movement.

Thanks for answering, good luck with this paper!

**Comment by liuyuan-pal** [comment]
Hi all,
1. The actual batch size is 2 on 80G GPU, we use 8 GPU and accumulate 4 times to get an effective batch size of 64. Our codes are based on CogVideoX and there are some repos to save GPU memories.
2. If the depth scale is incorrect, then the objects would be distorted. It's easy to use recent metric depth estimators Metric3D, Depth Pro, MoGE.
3. A too-large viewpoint change will cause all tracking points out of visible range, which leads to uncontrolled generation as shown in the limitations. It would be a interesting direction to exploit this technique for large scale scene generation.

**Comment by CorradoF** [comment]
For the motion transfer What is the minimun vram for inference and what's approximately the speed per frame?

**Comment by haodong2000** [comment]
Hi authors, very impressive work!!!

Will the training code also be released? Can't wait to fine-tune the video diffusion model on new tasks!!!

Best,
Haodong

**Comment by Njasa2k** [comment]
> Will the training code also be released? 

@liuyuan-pal

**Comment by MinJunKang** [comment]
Thanks for sharing the code. Now I will close the issue
